containers/containers.go

```diff
@@ -27,7 +29,10 @@ type RuntimeInfo struct {
 
 type Store interface {
 	Get(ctx context.Context, id string) (Container, error)
-	List(ctx context.Context, filter string) ([]Container, error)
+
+	// List returns containers that match one or more of the provided filters.
```

Comment:
What if I want only something that matches all the filters provided?

Should we add a boolean to decide whether it should be `All` or `Any` ?